murgatt/recode-converter
# feat: remove useless start screen

## Description
Remove the start page because it's useless.
Directly display the main converter page instead.

## How to test
- start the app
- you should not see the start page anymore but the converter page

## Checklist
